pytorch_lightning/trainer/training_loop.py

```diff
@@ -824,7 +825,7 @@ def training_step_and_backward(self, split_batch, batch_idx, opt_idx, optimizer,
                 self.warning_cache.warn("training_step returned None if it was on purpose, ignore this warning...")
                 return None
 
-            if self.trainer.train_loop.automatic_optimization:
+            if not self._skip_backward and self.trainer.train_loop.automatic_optimization:
```

Comment:
should we just return if we skip it ?

Reply:
What do you mean? this is not in a loop. After the if resolves to False it will return